server/graphql/Cargo.toml

```diff
@@ -21,6 +21,7 @@ graphql_cold_chain = { path = "cold_chain" }
 graphql_asset_catalogue = { path = "asset_catalogue" }
 graphql_inventory_adjustment = { path = "inventory_adjustment" }
 graphql_item_variant = { path = "item_variant" }
+graphql_item_bundle = { path = "item_bundle" }
```

Comment:
I went with `item_bundle` to keep it visually next to `item_variant` but I haven't been consistent with that.
Should this just be part of `item_variant` or should it be renamed??